docs/assets/0123-import-scripts/README.md

```diff
@@ -0,0 +1,142 @@
+shouldn't do is selecting a virtual environment of another project.
+
+![python interpreter](assets/python-interpreter.png)
+
+
+## The benefits of the debugging mode
+To start the debugging process, switch to the "Run and Debug" tab.
+
+1. set a break point
+2. click "Run and Debug"
+3. choose "Debug the currently active Python file"
+4. The control bar appears, and debugging starts.
+
+![configure debugging](assets/configure-debugging.png)
+
+Code execution will interrupt at your break point, that means, before the line of the break point is executed. Use 
+this opportunity to inspect what has been done until now in the "Variables" area on the left, where the current state of
+the program is shown.  
+
+If one of the dependencies is not installed, install it with `pip install package` in the Terminal of Visual Studio 
+Code.
+
+![pause at breakpoint](assets/pause-at-breakpoint.png)
+
+If you want to experiment with different scenarios how to proceed, go to the "Debug Console" where you can execute 
+code. For example, let's inspect the Pandas Dataframe by typing `main_df.info`.  
+You see that there are some empty rows at the end which don't contain useful data. The next two lines of code will 
+eliminate them. Click on "Step Over" two times, or set a new break point two lines further down and click on "Continue".
+Now, type again `main_df.info` in the Debug Console. You will see that the empty rows are gone.  
+
+![inspect dataframe](assets/inspect-dataframe.png)
+
+
+You see that the debugging mode is a useful tool to understand code and to inspect it for correctness. 
+
+| **Tip**                                                                                                   |
+|-----------------------------------------------------------------------------------------------------------|
+| **Make regular use of the debugging mode to check if your code really does what you think it should do!** | 
+
+
+## The benefits of version control
+One of the big benefits of version control is the diff viewer. Visual Studio highlights the changes you have introduced 
+since your last commit. 
+
+- Deletions are shown as red triangle.
+- Additions are shown as green bars.
+- Changed lines are shown as striped bars.
+
+Click on these visual elements to see a small popup that shows you the difference. In the popup, you can stage the 
+change, revert it, or jump to the next/previous change.
+
+![diffs](assets/diffs.png)
+
+Once you have a bunch of code changes that can be meaningfully grouped together, you should make a commit (and perhaps
+push it to a GitHub repo).
+
+
+| **Tips**                                                                |
+|-------------------------------------------------------------------------|
+| **Test your code (e.g. with the debugging mode) before committing it.** | 
+| **Make small commits that contain only one new feature.**               | 
```

Comment:
this section seems to address a user that does not know Git beforehand. But I think for that it's not extensive enough - I think it would make sense to at least add a link to a Git-introduction. Also, I hinted it before: I think Github Desktop is way more user friendly than the Git integration in VS Code, so if the audience is not familiar with Git at all, they'd have an easier time with Github Desktop